kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/pages/ConnectorUpdateView.java

```diff
@@ -0,0 +1,29 @@
+package com.provectus.kafka.ui.pages;
+
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+
+import static com.codeborne.selenide.Selenide.$;
+import static org.openqa.selenium.Keys.*;
+
+public class ConnectorUpdateView {
+    @Step
+    public ConnectorUpdateView updateConnectorConfig(String configJson) {
+        String os = System.getProperty("os.name");
+        System.out.println(os);
+        if (os.equals("MacOS")){
```

Comment:
you're using COMMAND in other lines as well, are you sure, you need `if` only for the first one`sendKeys`?